pkg/cmd/server/api/v1/types_test.go

```diff
@@ -157,6 +157,11 @@ kubernetesMasterConfig:
         location: ""
     pluginOrderOverride:
     - plugin
+    etcdClientInfo:
```

Comment:
this is nested incorrectly. run `hack/test-go.sh pkg/cmd/server/api/v1` manually to correct